js/alumni.js

```diff
@@ -232,10 +250,27 @@ function initMap() {
             title: college.name,
         });
         marker.setMap(map);
+        const contentString =
+        '<div id="content">' +
+        '<h1 id="collegeName" class="firstHeading">'+college.name+'</h1>' +
+        '<img id="collegeImg" src ='+college.image+'>'+
+        "</div>";
+        const infowindow = new google.maps.InfoWindow({
+            content: contentString,
+          });
         google.maps.event.addListener(marker,'click',function(event){  
             map.setCenter(marker.getPosition());  
-            map.setZoom(15);
-           
+            map.setZoom(6);   
+            infowindow.open({
+                anchor:marker,
+                map
+            });
+        });
+       
+        google.maps.event.addListener(map,'click',function(event){  
```

Comment:
Seems like something might be up with the indentation of this line, also use spaces after , and between ) and {